GPU/Common/FragmentShaderGenerator.h

```diff
@@ -39,4 +39,9 @@ struct FShaderID;
 // For stencil upload
 #define CONST_PS_STENCILVALUE 11
 
-bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLanguageDesc &compat, Draw::Bugs bugs, uint64_t *uniformMask, std::string *errorString);
+// Can technically be deduced from the fragment shader ID, but this is safer.
+enum FragmentShaderFlags {
+	FS_FLAG_INPUT_ATTACHMENT,
```

Comment:
I haven't looked at the other pull yet, but isn't this 0?  Shouldn't we have a `= 1` or NONE?

-[Unknown]

Reply:
Ah, this is fixed in the other PR that actually makes use of it. This is just a buildfix from a rebase, trying to tease the code apart...